.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Description
+<!---
+  Provide a general summary of your changes in the Title above
+  Describe your change(s) in detail here
+
+  Also add the relevant label in the column on the right:
+    Breaking changes: kind/breaking-change
+    New features:     kind/product-feature
+    Bug fixes:        kind/bug
+    Dependencies:     kind/dependencies
+    Other changes:    kind/other
```

Comment:
Dere må dele ut `Triage` rettigheter før eksterne kan gjøre dette.

![image](https://user-images.githubusercontent.com/131616/210242322-59bf1ddf-7b77-4a6e-86b0-0acfaee96ca5.png)

Reply:
@altinnadmin Kan vi gi Triage-rettigheter til `ext-*`? [Ser ut til at det gir mulighet til å lukke issues, legge på/endre labler osv](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role). Om ikke kan vi jo like greit bare endre litt i ordlyden og skrive at eksterne bør overlate labling til oss.